### PR TITLE
prank 150803

### PR DESCRIPTION
**Pull request by tseemann**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [x] Removed the `revision` line, if any, when bumping the version number?
- [x] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [x] Not altered the `bottle` section?
- [x] Checked that the tests still pass?

### CODE REVIEW

**Comment by tseemann** [comment]
why does this always fail for me?

```
% brew pull --bottle https://github.com/Homebrew/homebrew-science/pull/5301

==> Fetching patch
Patch: https://github.com/Homebrew/homebrew-science/pull/5301.patch
==> Applying patch
Applying: prank 150803
==> Patch closes issue #5301
==> Fetching patch (bottle commit)
Patch: https://github.com/BrewTestBot/homebrew-science/compare/homebrew:master...pr-5301.patch
==> Applying patch
Applying: prank 150803
Applying: prank: update 150803 bottle.
==> Publishing on Bintray: prank 150803

Error: Failure while executing: /usr/bin/curl --remote-time --location --user-agent Homebrew/1.1.11-132-g17cc40f11\ (Macintosh;\ Intel\ macOS\ 10.12.3)\ curl/7.51.0 --progress-bar --fail -w \n --silent --fail -utseemann:738694405ca36848289335a41de9034b53641696 -X POST -H Content-Type:\ application/json -d {"publish_wait_for_secs":\ 0} https://api.bintray.com/content/homebrew/bottles-science/prank/150803/publish
```

**Comment by tseemann** [comment]
@sjackman i don't have homebrew-science permissions anymore.

**Comment by sjackman** [comment]
I messaged you on Slack.

**Comment by iMichka** [comment]
@BrewTestBot test this please

**Comment by sjackman** [comment]
Thanks, Torsten! I think the DOI for PRANK is the 2008 article https://doi.org/10.1126/science.1158395 or possibly the 2004 article https://doi.org/10.1073/pnas.0409137102 Would you like to add one of those?

**Comment by ilovezfs** [comment]
Current version is now 170427.

File: prank.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 class Prank < Formula
-  desc "A multiple alignment program for DNA, codon and amino-acid sequences"
+  desc "Codon-aware multiple sequence alignment"
   homepage "http://wasabiapp.org/software/prank/"
-  url "http://wasabiapp.org/download/prank/prank.source.140603.tgz"
-  sha256 "9a48064132c01b6dba1eec90279172bf6c13d96b3f1b8dd18297b1a53d17dec6"
+  # tag "bioinformatics"
```
Comment (sjackman) [inline prank.rb]: Can you please move this comment after `head` and add a `# doi`?

**Pull request closed without merging**